includes/data/actions.lua

```diff
@@ -387,7 +387,7 @@ function entToNet(netID)
     end
     if NETWORK.NETWORK_HAS_CONTROL_OF_NETWORK_ID(netID) then
       NETWORK.SET_NETWORK_ID_CAN_MIGRATE(netID, true)
-      NETWORK.SET_NETWORK_ID_EXISTS_ON_ALL_MACHINES(netID)
+      NETWORK.SET_NETWORK_ID_EXISTS_ON_ALL_MACHINES(netID, false)
```

Comment:
Not sure if this is correct because p2 is called "toggle" in NativeDB.
But this is what it was doing before.

Reply:
It was supposed to be `true` but it doesn't matter either way because it's unnecessary. I will get rid of it later.